app-dashboard/sql/flood_event_sub_district_summary.sql

```diff
@@ -0,0 +1,184 @@
+--        avoid faulty rows (where the id's are null)
+               where district_id is not null
+                 and sub_district_id is not null
+                 and village_id is not null) as a
+--  aggregate by sub district and flood_event_id
+         group by district_id, sub_district_id, flood_event_id
+     ),
+     flooded_aggregate_count as (
+-- combines flooded count information and total building information
+         select a.flood_event_id,
+                a.district_id,
+                a.sub_district_id,
+                a.flooded_building_count,
+                b.building_count,
+                b.residential_building_count,
+                b.clinic_dr_building_count,
+                b.fire_station_building_count,
+                b.school_building_count,
+                b.university_building_count,
+                b.government_building_count,
+                b.hospital_building_count,
+                b.police_station_building_count,
+
+                a.residential_flooded_building_count,
+                a.clinic_dr_flooded_building_count,
+                a.fire_station_flooded_building_count,
+                a.school_flooded_building_count,
+                a.university_flooded_building_count,
+                a.government_flooded_building_count,
+                a.hospital_flooded_building_count,
+                a.police_station_flooded_building_count,
+                a.total_vulnerability_score
+         from flooded_count_selection as a
+                  join non_flooded_count_selection b
+                       on a.district_id = b.district_id and a.sub_district_id = b.sub_district_id
+     )
+
+select flooded_aggregate_count.flood_event_id,
+       flooded_aggregate_count.district_id,
+       flooded_aggregate_count.sub_district_id,
+       flooded_aggregate_count.building_count,
+       flooded_aggregate_count.flooded_building_count,
+       flooded_aggregate_count.total_vulnerability_score,
+       flooded_aggregate_count.residential_flooded_building_count,
+       flooded_aggregate_count.clinic_dr_flooded_building_count,
+       flooded_aggregate_count.fire_station_flooded_building_count,
+       flooded_aggregate_count.school_flooded_building_count,
+       flooded_aggregate_count.university_flooded_building_count,
+       flooded_aggregate_count.government_flooded_building_count,
+       flooded_aggregate_count.hospital_flooded_building_count,
+       flooded_aggregate_count.police_station_flooded_building_count,
+
+       flooded_aggregate_count.residential_building_count,
+       flooded_aggregate_count.clinic_dr_building_count,
+       flooded_aggregate_count.fire_station_building_count,
+       flooded_aggregate_count.school_building_count,
+       flooded_aggregate_count.university_building_count,
+       flooded_aggregate_count.government_building_count,
+       flooded_aggregate_count.hospital_building_count,
+       flooded_aggregate_count.police_station_building_count
```

Comment:
We are missing the islam, buddist counts both for flooded and non flooded

Reply:
I omit those for now with intention because, it seems a sensitive issue where we only include two religions (from many).
I knew the reason was to use these place as evacuation center too.
But I need to confirm with PMI first.

Other than that, we might need to fix selection for clinic and university. It seems to not match nicely.